dotnet-471-dev-pack/test/test.ps1

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env pwsh
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Validate Input Parameters and System Software
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$PackageIdentifier = "",
+    [ValidateScript({
+        if( -Not ($_ | Test-Path) ){
+            throw "File does not exist"
+        }
+        if($_ -notmatch "(\.hart)"){
+            throw "The file specified in the path argument must be of type *.hart"
+        }
+        return $true
+    })]
+    [System.IO.FileInfo]$PathToHartArtifact
+)
+if(-Not ( $PathToHartArtifact | Test-Path) ){
+    throw "-PathToHartArtifact must a string and set to a valid path to a *.hart artifact"
+}
+
+# validate pre-requisite software on test system
+if (-Not (Get-Module -ListAvailable -Name Pester)) {
+    $message = -join @(
+        "Pester module must be installed before proceeding. "
+        "Try something like the following and then re-run: "
+        "'Install-Module Pester -SkipPublisher -Force'"
```

Comment:
These flags might not be needed and it might not be ideal for us to advise accepting unsigned code and force installing a module.
We can just give the URL https://github.com/pester/Pester/wiki/Installation-and-Update and ask the reader to follow that.

Reply:
Adjusted comment to reference the Pester URL.